assets/js/modules/ads/pax/services.test.js

```diff
@@ -267,5 +267,16 @@ describe( 'PAX partner services', () => {
 				} );
 			} );
 		} );
+
+		describe( 'termsAndConditionsService', () => {
+			it( 'notify callback should return an empty object', async () => {
```

Comment:
The hierarchy has gotten off here, although it works. Let's correct it for consistency.

It should have
```
- describe service
- - describe method
- - - it ...
```

- `termsAndConditionsService` is missing the inner describe for `notify`
- `getSupportedConversionTrackingTypes` is out of place (not nested under its service)

Reply:
Addressed in latest commit thanks @aaemnnosttv